test/functional/feature_evm_contracts.py

```diff
@@ -438,6 +438,21 @@ def non_payable_proxied_contract(self):
             receipt["gasUsed"] * receipt["effectiveGasPrice"],
         )
 
+    # should fail since it is larger than DVM TX size limit
+    def fail_send_large_tx(self):
+        assert_raises_web3_error(
+            -32001,
+            "reason: scriptpubkey",
+            self.node.w3.eth.send_transaction,
+            {
+                "to": "0x0000000000000000000000000000000000000000",
+                "from": self.evm_key_pair.address,
+                "data": "0x" + "ff" * 64000,
```

Comment:
The failure is not due to size. This will fail regardless of the size. Will fail even if it's just set to ff.

Reply:
Updated in https://github.com/DeFiCh/ain/pull/2652/commits/9dca6dc55da3269030d7a02e3fbba2aed07c862d